src/main/javascript/src/components/ui/Mark/Mark.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Icon } from 'antd';
+
+import './mark.scss';
+
+const BLOCK = 'ui_mark';
+
+interface MarkProps {
+  mark: 0 | 1 | 2 | null;
+}
+
+export const Mark: React.FunctionComponent<MarkProps> = ({ mark }) => (
+  <div className={BLOCK}>
+    <Icon className={`${BLOCK}__icon`} type="star" theme={mark ? 'filled' : undefined} />
```

Comment:
No bones ? :cry:

Reply:
I didn't find them pre-built so we'll have to change them in another PR 😉